### PR TITLE
Kubedash bundled with old Heapster (v18)

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @piosz @fgrzadkowski

### CODE REVIEW

**Comment by fgrzadkowski** [comment]
I believe we will also need some explanation why we need it in the first place.

**Comment by mwielgus** [comment]
@fgrzadkowski sure. Readme.md updated as well.

File: README.md [hunk 1]
```diff
@@ -20,12 +20,18 @@ Future features include the following:
 
 To use Kubedash, the following are required:
 - A Kubernetes cluster of v1.0 or higher is operational and accessible from kubectl.
-- Heapster v0.18.0 is running as the `heapster` service on the `kube-system` namespace.
+- For Kubernetes <1.2 Heapster v0.18.0 is running as the `heapster` service on the `kube-system` namespace.
```
Comment (vishh) [inline README.md]: nit: Was this meant to be `>= v1.2`?

File: README.md [hunk 2]
```diff
@@ -20,12 +20,18 @@ Future features include the following:
...
 
 After cloning this repository, use the following command to create the Kubedash pod:
 
+For Kubernets 1.0 and 1.1:
+
 `kubectl create -f deploy/kube-config.yaml`
 
+For Kubernetes 1.2 and higher:
+
+`kubectl create -f deploy/bundle.yaml`
```
Comment (vishh) [inline README.md]: Has this new yaml been added to the kubedash repo?

File: README.md [hunk 3]
```diff
@@ -39,6 +45,9 @@ The [Heapster Model](https://github.com/kubernetes/heapster/blob/master/docs/mod
 
 Kubedash provides the other two pieces - a web server relaying REST calls, managing sockets and providing additional authentication and, a frontend to provide visualizations for the aggregated metrics and statistics of interest.
 
+Heapster in Kubernetes 1.2 is not compatibile with the current version of Kubedash. For Kuberentes 1.2 we suggest deploying Kubedash
+with a custom Heapster version running inside a single Pod. See [this](deploy/bundle.yaml) for details.
```
Comment (vishh) [inline README.md]: nit: It is not a `custom` version right. It is just an older version.

File: README.md (hunk 2 above)
Comment (mwielgus) [inline README.md]: It is in this PR.

File: README.md (hunk 1 above)
Comment (mwielgus) [inline README.md]: No. For >= 1.2 we don't care whether there is any Heapster or no. There is one in the bundle.

File: README.md (hunk 2 above)
Comment (vishh) [inline README.md]: Well, this information needs to exist in the kubedash repo as well right?

File: deploy/bundle.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+  apiVersion: "v1"
+  kind: "ReplicationController"
```
Comment (vishh) [inline deploy/bundle.yaml]: Since this is a new config, can we make it a deployment instead?

File: deploy/bundle.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+              - "--heapster_url=http://localhost:8082"
+            resources:
+              limits:
+                cpu: 50m
+                memory: 100Mi
+            volumeMounts:
+              - name: "ssl-certs"
+                mountPath: "/etc/ssl/certs"
+                readOnly: true
+          - image: gcr.io/google_containers/heapster:v0.18.4
```
Comment (vishh) [inline deploy/bundle.yaml]: nit: Why not v0.19?

File: README.md (hunk 1 above)
Comment (vishh) [inline README.md]: Ahh.. Sorry I mis-read the statement..

File: deploy/bundle.yaml (hunk 5 above)
Comment (mwielgus) [inline deploy/bundle.yaml]: Because it has  never been released as a part of Kubernetes (and is probably not tested enough).

**Comment by vishh** [comment]
Depending on when the dashboard integrates some of the kubedash functionalities, we might want to port kubedash to the new heapster API. Can we try to get some help from the community for that effort?

File: README.md (hunk 2 above)
Comment (mwielgus) [inline README.md]: I don't understand. The new file (deployment/bundle.yaml) is in the PR.

**Comment by mwielgus** [comment]
Sure, if you have time to drive this feel free to ask for community help.

File: deploy/bundle.yaml (hunk 4 above)
Comment (mwielgus) [inline deploy/bundle.yaml]: Ok.

File: README.md (hunk 3 above)
Comment (mwielgus) [inline README.md]: Done.

File: deploy/bundle.yaml (hunk 4 above)
Comment (mwielgus) [inline deploy/bundle.yaml]: Done.

File: README.md (hunk 2 above)
Comment (vishh) [inline README.md]: :+1: While reviewing on phone, I overlooked the fact that this PR was in the kubedash repo.. Sorry for all the confusion..

File: deploy/bundle.yaml (hunk 5 above)
Comment (vishh) [inline deploy/bundle.yaml]: Even then it has to be v0.18.5 right? Din't we make a patch release with some critical bug fixes?

Comment (mwielgus) [inline deploy/bundle.yaml]: done

**Comment by vishh** [comment]
Testing this patch now.

**Comment by vishh** [comment]
LGTM. Thanks @mwielgus !!

**Pull request merged**